changelogs/fragments/397-iso_create.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - iso_create - add new iso_create.py module file in files folder
```

Comment:
```suggestion
  - iso_create - add new iso_create module (https://github.com/ansible/ansible/issues/35665).
```

Reply:
New modules must have no changelog fragment. Simply remove this file :)

Reply:
ah, forgot, exactly, thanks!

Reply:
Ok, I'll delete it, thanks.